shared/chat/conversation/input-area/normal/platform-input.desktop.js

```diff
@@ -279,6 +292,36 @@ class PlatformInput extends Component<PlatformInputProps, State> {
                   type="iconfont-boom"
                 />
               )}
+            {flags.explodingMessagesEnabled &&
+              this.state.explodingPopupOpen && (
+                <SetExplodingMessagePopup
```

Comment:
This is close to needing to be factored into a connected component. But it can probably wait until the `isNew` flag integration